src/api.js

```diff
@@ -1,12 +1,28 @@
 import axios from "axios";
-import {BACKEND_BASE_URI, COMPANIES_API_PATH} from "./constants";
+import { BACKEND_BASE_URI, COMPANIES_API_PATH } from "./constants";
 
 /*
   Fetch list of all companies from backend
  */
 export const fetchCompanies = async () => {
   try {
-    const response = await axios.get(`${BACKEND_BASE_URI}${COMPANIES_API_PATH}`);
+    const formatter = new Intl.NumberFormat("en-US", {
+      style: "currency",
+      currency: "USD",
+    });
+
+    const dateFormatter = new Intl.DateTimeFormat("en-US", {
```

Comment:
Same problem as in the situation with revenue. You solved the different problem - not the one described in the issue. For example - your solution converts ISO 8601 date into 08:41 PM. The description of the issue asks to convert the same date to 20:41.

Please update the implementation to match the expected behavior as explained in the issue.